content/post/078-go-ipfs-0.4.23.md

```diff
@@ -0,0 +1,84 @@
+---
+date: 2020-01-27
+url: 2020-01-27-go-ipfs-0-4-23
+title: go-ipfs 0.4.23 released
+author: Alan Shaw
+---
+
+> Would sir/madam care for another patch release while they wait?
+
+Yes that's right, the next feature release of go-ipfs (0.5.0) is, well, running a tiny bit behind schedule. In the mean time though we have patches, and I'm not talking pirate eye patches, I'm talking bug fixes. We're hunting these bugs like they're Pokemon, and jeez, do we come across some rare and difficult to fix ones? - you betcha.
+
+Alright, enough funny business, what's the deal? Ok so, I don't want to _alarm_ anyone but this release has some **critical fixes** and if you're using go-ipfs or know someone who is then you and your friends need to slide into your upgrade pants and give those IPFS nodes a good wipe down ASAP.
```

Comment:
In terms of critical fixes, we should explicitly highlight (up front):

1. It fixes the websocket crasher.
2. It fixes an issue in the TLS transport _and_ TLS will be enabled by default in go-ipfs 0.5.0.